numpy/core/tests/test_multiarray.py

```diff
@@ -2091,6 +2097,14 @@ def test_searchsorted_with_sorter(self):
             assert_equal(b, out)
             b = a.searchsorted(a, 'r', s)
             assert_equal(b, out + 1)
+            # Test empty array, use a fresh array to get warnings in
```

Comment:
What would happen if you constructed the empty array once here, and reused it?

Reply:
I've gone ahead and changed that - it makes the test substantially easier to read